data/mods/gen3/moves.ts

```diff
@@ -413,7 +413,8 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 			},
 			onDamagePriority: -101,
 			onDamage(damage, target, source, effect) {
-				if (effect.effectType === 'Move' && !source.isAlly(target) && this.getCategory(effect.id) === 'Special') {
+				if (effect.effectType === 'Move' && !source.isAlly(target) &&
+				this.getCategory(effect.id) === 'Special' && effect.id !== 'weatherball') {
```

Comment:
```suggestion
				if (
					effect.effectType === 'Move' && !source.isAlly(target) &&
					this.getCategory(effect.id) === 'Special' && effect.id !== 'weatherball'
				) {
```
Really wish eslint could enforce this :s